src/Device/Driver/LX_EOS/LXEosDevice.hpp

```diff
@@ -259,11 +335,47 @@ class LXEosDevice : public AbstractDevice
    * @param env OperationEnvironment
    * @return true if ACK was received,
    * @return false if time out or other exception occurred
-   * @note timeout is 1 second
+   * @note timeout is 3 seconds
    */
   bool WriteAndWaitForACK(const std::span<std::byte>& message,
                           OperationEnvironment& env);
 
+  /**
+   * @brief Get the Number Of Flights in logger. Maximum number that can be
+   * read is 0xFF
+   *
+   * @param flight_count output number of flights (max 0xFF)
+   * @param env
+   * @return true if successful
+   * @note(flight_count not changed on unsuccess)
+   */
+  bool GetNumberOfFlights(uint8_t& flight_count, OperationEnvironment& env);
```

Comment:
Return the number of flights instead of passing a reference variable for the result.